packages/core/src/browser/style/menus.css

```diff
@@ -30,7 +30,7 @@
 |----------------------------------------------------------------------------*/
 
 
-.p-MenuBar {
+.p-Widget.p-MenuBar {
```

Comment:
So `p-MenuBar` is not used anymore?
What about this reference https://github.com/theia-ide/theia/blob/89c3e75f4a75f5850b7797ad6af3ca8ac3700149/packages/core/src/browser/style/menus.css#L60

Reply:
> So p-MenuBar is not used anymore?
Only in `p-Widget` container. Are there other containers where it maybe could be used in?

> What about this reference
Since this rule is not affected by the p-Widget rule I left it out.